.github/workflows/test.yml

```diff
@@ -8,7 +8,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        go: [1.16.x, 1.17.x]
+        go: [1.17.x, 1.18.x]
```

Comment:
Oh, I guess this means 1.17 should be dropped entirely. @FiloSottile, are you ok with this?